tools/xcodeproj_shims/install.sh

```diff
@@ -10,6 +10,13 @@
 set -eux
 
 
+diagnostics_dir="$TARGET_BUILD_DIR/../../../bazel-xcode-diagnostics/"
```

Comment:
will the files in here ever get cleaned up?

Reply:
I'll add code to delete older files and generate a new file for each run that includes the current timestamp in the filename.